numpy/polynomial/__init__.py

```diff
@@ -58,8 +58,8 @@
 The following lists the various constants and methods common to all of
 the classes representing the various kinds of polynomials. In the following,
 the term ``Poly`` represents any one of the convenience classes (e.g.
-``Polynomial``, ``Chebyshev``, ``Hermite``, etc.) while the lowercase ``p``
-represents an **instance** of a polynomial class.
+`~polynomial.Polynomial`, `~chebyshev.Chebyshev`, `~hermite.Hermite`, etc.)
+while the lowercase ``p`` represents an **instance** of a polynomial class.
```

Comment:
I think I'd be inclined to revert the changes to these lines

Reply:
It was actually those cases that I wanted to click on. Why not make them links, given that the text did not change?

Reply:
I think the intention is that only the first appearance of a keyword should be a link, or maximum the first appearance in each section. Making every appearance a link is distracting.

Reply:
@mattip Point taken. I think I've removed all the undesirable links. Please let me know if I missed any.

Reply:
The original intent here was to highlight the naming convention used for the remainder of the module docstring - i.e. that ``Poly`` will be used as a stand-in for any one of the 6 convenience classes in the following listings.

While the `~<module>.<class>`-style linking works fine in the rendered html, I think it makes the source rst harder to parse in this instance. I tend to agree with @eric-wieser & @mattip that I prefer the original, and having the links to the convenience class in the table that appears at the top makes the most sense.